README.md

```diff
@@ -34,6 +33,58 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Mandatory requirements
+
+  * Python 2.7 (the code here is not Python 3.x ready)
+  * bzip2 / bunzip2
+  * Python modules: numpy, rpy2
+  * R
+
+### Optional requirements
+
+  * Python modules required for plotting: matplotlib, seaborn
+  * Required for generating LaTeX tables: pdflatex
```

Comment:
Should we say "a latex distribution containing pdflatex and the x, y, z packages. TeX Live works fine."